keyboards/bm68rgb/bm68rgb.c

```diff
@@ -18,42 +18,27 @@
-    // LShift, Z, X, C, V, B, N, M, ,, ., /, Shift, Up, Page Down
-                   {  9,  48 }, {  34,  48 }, {  49,  48 }, {  64,  48 }, {  79,  48 }, {  94,  48 }, { 109,  48 }, { 124,  48 }, { 139,  48 }, { 154,  48 }, { 169,  48 }, { 189,  48 }, { 210,  48 }, { 225,  48 },
-    // Ctrl, GUI, Alt, Space, RAlt, FN, Ctrl, Left, Down, Right
-    {   2,  64 }, {  21,  64 }, {  39,  64 },                                { 94,  64 },                                         { 148,  64 }, { 163,  64 }, { 178,  64 }, { 193,  64 }, { 208,  64 }, { 225,  64 },
-    // UNDERGLOW
-    {   185, 45 }, {  160,  45 }, {  125, 45 }, { 95,  45 }, { 60,  45 }, { 25,  45 }
+	// LED Index to Physical Position
+	{  0,  0}, { 15,  0}, { 30,  0}, { 45,  0}, { 60,  0}, { 75,  0}, { 90,  0}, {105,  0}, {120,  0}, {135,  0}, {150,  0}, {165,  0}, {180,  0}, {202,  0}, {225,  0}, // Esc, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, -, =, Backspace, Delete
+	{  4, 16}, { 22, 16}, { 37, 16}, { 52, 16}, { 67, 16}, { 82, 16}, { 97, 16}, {112, 16}, {127, 16}, {142, 16}, {157, 16}, {172, 16}, {187, 16}, {206, 16}, {225, 16}, // Tab, Q, W, E, R, T, Y, U, I, O, P, [, ], backslash , Home
+	{  6, 32},            { 26, 32}, { 41, 32}, { 56, 32}, { 71, 32}, { 86, 32}, {101, 32}, {116, 32}, {131, 32}, {146, 32}, {161, 32}, {176, 32}, {201, 32}, {225, 32}, // Capslock, A, S, D, F, G, H, J, K, L, ;, ', Enter, Page up
+	           {  9, 48}, { 34, 48}, { 49, 48}, { 64, 48}, { 79, 48}, { 94, 48}, {109, 48}, {124, 48}, {139, 48}, {154, 48}, {169, 48}, {189, 48}, { 10, 48}, {225, 48}, // LShift, Z, X, C, V, B, N, M, ,, ., /, Shift, Up, Page Down
+	{  2, 64}, { 21, 64}, { 39, 64},                                  { 94, 64},                       {148, 64}, {163, 64}, {178, 64}, {193, 64}, {208, 64}, {225, 64}, // Ctrl, GUI, Alt, Space, RAlt, FN, Ctrl, Left, Down, Right
+	{185, 45}, {160, 45}, {125, 45}, { 95, 45}, { 60, 45}, { 25, 45} // UNDERGLOW
 }, {
-    // LED Index to Flag
-    //need to recheck what should be a modifier
-
-    // Esc, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, -, =, Backspace, Delete
-    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 1,
-    // Tab, Q, W, E, R, T, Y, U, I, O, P, [, ], backslash , Home
-    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
-    // Capslock, A, S, D, F, G, H, J, K, L, ;, ', Enter, Page up
-    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 1,
-    // LShift, Z, X, C, V, B, N, M, ,, ., /, Shift, Up, Page Down
-    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 1, 1,
-    // Ctrl, GUI, Alt, Space, RAlt, FN, Ctrl, Left, Down, Right
-    1, 1, 1, 4, 1, 1, 1, 1, 1, 1,
-    // UNDERGLOW
-    2, 2, 2, 2, 2, 2
+	// LED Index to Flag
+	1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 1,  // Esc, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, -, =, Backspace, Delete
+	1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,  // Tab, Q, W, E, R, T, Y, U, I, O, P, [, ], backslash , Home
+	1,    4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 1,  // Capslock, A, S, D, F, G, H, J, K, L, ;, ', Enter, Page up
```

Comment:
Consider changing Capslock to 9 (1+8)

Reply:
This proposal might be _too specific_ to a single use case. Many folks do not use a dedicated caps lock key. It is actually occupying a prime position on the keyboard. That spot is frequently replaced by back space or Ctrl key.